recipes/clhep/all/conanfile.py

```diff
@@ -6,8 +6,7 @@
 from conan.tools.microsoft import is_msvc
 import os
 
-required_conan_version = ">=1.53.0"
-
+required_conan_version = ">=2.0.9"
```

Comment:
Is it allowed by cci policy? I don't see any value to break conan v1 users.

Reply:
it doesn't break conan v1 users, they have available the previous recipe revisions still (don't they?). They just won't get the new version. But, for example, given that "implements" attribute is only available in conan2 that's why I bumped the required conan version. I wasn't aware that use of conan 2 only features was disallowed but I do see now in FAQ it states things should be 1 and 2 compatible so I will revert this

Reply:
It will break conan v1 users if they want to update recipe RREV. `implements` avoids some boilerplate, it's not worth breaking v1 users.

For the moment, recipes must support conan v1, see https://github.com/conan-io/conan-center-index/discussions/19104#discussion-5500696